spec/presenter_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'presenter'
+require 'board'
+
+class FakeIO
```

Comment:
If you're using the same fake in multiple places, maybe it's worth extracting them out into a separate class in the `spec/` directory.